common/web/types/src/util/util.ts

```diff
@@ -19,12 +19,47 @@ export function boxXmlArray(o: any, x: string): void {
 // TODO-LDML: #7569 the below regex works, but captures more than it should
 // (it would include \u{fffffffffffffffff } which
 // is overlong and has a space at the end.) The second regex does not work yet.
-const MATCH_HEX_ESCAPE = /\\u{([0-9a-fA-F ]{1,})}/g;
+export const MATCH_HEX_ESCAPE = /\\u{([0-9a-fA-F ]{1,})}/g;
 // const MATCH_HEX_ESCAPE = /\\u{((?:(?:[0-9a-fA-F]{1,5})|(?:10[0-9a-fA-F]{4})(?: (?!}))?)+)}/g;
 
+/** regex for single quad escape such as \u0127 */
+export const MATCH_QUAD_ESCAPE = /\\u([0-9a-fA-F]{4})/g;
```

Comment:
Will this match the alternate escaping pattern referred to in the user test?  it doesn't seem like this regex would match `'\u{0127}'`, but it's a perfectly valid JS quad-escape, isn't it?

Reply:
No that is handled by the previous escape. This is for a special one that we might remove with no curly braces